src/amo/containers/CategoryPage.js

```diff
@@ -0,0 +1,41 @@
+import { connect } from 'react-redux';
+import { asyncConnect } from 'redux-connect';
+import { compose } from 'redux';
+
+import SearchPage from 'amo/components/SearchPage';
+import {
+  filtersMatch,
+  loadCategoryResultsIfNeeded,
+  parsePage,
+} from 'core/searchUtils';
+
+
+export function mapStateToProps(state, ownProps) {
+  const filters = {
```

Comment:
Ah, ok, I see on a second look that all of these do need to be defined outside of the if block.